node/src/components/consensus/highway_core/active_validator.rs

```diff
@@ -77,12 +83,14 @@ impl<C: Context> ActiveValidator<C> {
         timestamp: Timestamp,
         state: &State<C>,
     ) -> (Self, Vec<Effect<C>>) {
+        let next_round_exp = state.params().init_round_exp();
```

Comment:
We're still using the [minimum round exponent as the initial one](https://github.com/CasperLabs/casper-node/blob/1b734719dbecf65af64afc6132da41f23eb4ad0a/node/src/components/consensus/highway_core/state/params.rs#L49) (my bad). With short eras, that will force us to almost always go with the minimum one.

We could e.g. have a rule like:
If the previous era was a compatible Highway instance, use the median of round exponents as seen by the switch block. Otherwise, use the minimum.

(But maybe that's work for a separate PR/ticket.)

Reply:
This is now ticketed, so I'll resolve this conversation.